blocks/textarea/textarea.css

```diff
@@ -7,12 +7,16 @@
   border-radius: 1px;
   outline: none;
   resize: none;
-  overflow: hidden;
-  font-family: "Open Sans", sans-serif; 
+  font-family: "OpenSans", arial, sans-serif; 
   color: rgb(40, 54, 69); 
   font-size: 18px;
-  font-weight: 400;
-  line-height: 30px;
+  line-height: 166.667%;
```

Comment:
It is better to set line-height in fractional numbers instead of percentages. In your case, it will look like this:
`line-height: 1.66;`